### PR TITLE
Make FieldtypeOptions detect single value usage

### PR DESCRIPTION
**Pull request by LostKobrakai**
Make FieldtypeOptions detect single value usage from the inputfield with markupValue() usage

### CODE REVIEW

File: wire/modules/Fieldtype/FieldtypeOptions/FieldtypeOptions.module [hunk 1]
```diff
@@ -207,6 +207,10 @@ class FieldtypeOptions extends FieldtypeMulti implements Module {
 	 *
 	 */
 	public function ___markupValue(Page $page, Field $field, $value = null, $property = '') {
+		$interfaces = class_implements($field->inputfieldClass);
+		if(!in_array(InputfieldHasArrayValue::class, $interfaces) && $value instanceof WireArray) {
```
Comment (teppokoivula) [inline wire/modules/Fieldtype/FieldtypeOptions/FieldtypeOptions.module]: Just an observation: this requires PHP >= 5.5.0. Is that OK?

Comment (LostKobrakai) [inline wire/modules/Fieldtype/FieldtypeOptions/FieldtypeOptions.module]: I was worried about `class_implements`, but that's actually 5.1+. Didn't think of `::class`. Any better idea than hardcoding the string?

Comment (clsource) [inline wire/modules/Fieldtype/FieldtypeOptions/FieldtypeOptions.module]: What about http://php.net/manual/en/function.get-class.php ?

**Comment by ryancramerdesign** [comment]
Thanks @LostKobrakai this has been added (with minor changes).

**Pull request closed without merging**